test/kata5.reachDestination.test.js

```diff
@@ -1,5 +1,10 @@
 const { reachDestination } = require("../src");
 
 describe("reachDestination", () => {
-  test("returns string with estimated time of arrival", () => {});
+  test("returns string with estimated time of arrival", () => {
+    expect(reachDestination(44, 10)).toEqual("I should be there in 4.5 hours.");
```

Comment:
Could a test be added that returns a value for the hour that doesnt have a remainder here?